corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -409,7 +417,7 @@ def uses_managed_data_source(self):
     @property
     @memoized
     def data_source(self):
-        return get_datasource_config_infer_type(self.source_id, self.domain)[0]
+        return get_datasource_config_infer_type(self._source_id, self.domain)[0]
```

Comment:
I'm not following - why does this use `_source_id` instead of `data_source_id`?  Why have two names for it at all?  Is this to prevent it from being assigned outside `__init__`?

Reply:
probably best to just remove the @property and assign it directly to `data_source_id` in `__init__`

Reply:
64c5fec4491